.github/workflows/run_tests.yml

```diff
@@ -0,0 +1,96 @@
+name: Unit Tests
+
+on:
+  push:
+    branches-ignore:
+      - 'test_deploy'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  run_tester:
+    runs-on: ${{ matrix.config.os }}
+    if: "!contains(github.event.head_commit.message, 'ci skip')"
+    env:
+      RENV_PATHS_ROOT: ~/.local/share/renv
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+        - {name: '3.9', os: ubuntu-latest, python: '3.9', r: 'release'}
+        - {name: '3.8', os: ubuntu-latest, python: '3.8', r: 'release'}
+        - {name: '3.7', os: ubuntu-latest, python: '3.7', r: 'release'}
+        - {name: '3.6', os: ubuntu-latest, python: '3.6', r: 'oldrel'}
```

Comment:
Even 3.7 is old by now :) Do we really need 3.6? 3.10 might be more beneficial

Reply:
Point taken :)

Reply:
Though right now `pyproject.toml` does say anndata2ri supports py>=3.6, so I would argue if you support it, it should be tested

Reply:
All right :)
I'd argue that we should drop 3.6 support because the latest AnnData & scanpy don't support it either, but this might be beyond the scope of this PR :)

Reply:
I agree on both counts :)